Xamarin.Essentials/MediaPicker/MediaPicker.ios.cs

```diff
@@ -134,6 +157,11 @@ static FileResult DictionaryToMediaFile(NSDictionary info)
             return new PHAssetFileResult(assetUrl, phAsset, originalFilename);
         }
 
+        static FileResult PickerResultsToMediaFile(PHPickerResult[] results)
+        {
+            throw new NotImplementedException();
```

Comment:
TODO

Reply:
@jamesmontemagno @mattleibow @Redth This is implemented in #1750